random-forest/python/requirements.txt

```diff
@@ -0,0 +1,3 @@
+numpy==1.15.0
+scikit-learn==0.20.2
+sigopt_sklearn>=2.0.0
```

Comment:
I think @pfhayes was referring to pinning the sklearn version in line 2. We're not actually using the sigopt sklearn library here, so you only need to pin the version for the normal `sigopt` python library

Reply:
I believe it gets used by the other example in the folder?

https://github.com/sigopt/sigopt-examples/blob/master/random-forest/python/random_forest.sklearn.py

I could pin it as well, remove it entirely or create separate requirements for each example.

Since sigopt_sklearn requires sigopt, I'm not sure what best practices are in terms of whether I should also require sigopt in this if we keep sigopt_sklearn

Reply:
Ohhh I see. Well, to me it would be sense to include both `sigopt` and `sigopt_sklearn` in the folder, because one example uses one and other uses the other. Then the dependencies can be managed independently of one another

Reply:
Sure thing, added it in.